.github/workflows/leo-build-tag-publish-and-run-tests.yml

```diff
@@ -200,7 +200,7 @@ jobs:
 
       - name: dispatch to terra-github-workflows
         env:
-          leo_test_command: "testOnly -- -l ExcludeFromJenkins"
+          leo_test_command: "testOnly -- -l ProdTest -l NotebooksCanaryTest -l ExcludeInFiab -l ExcludeFromJenkins"
```

Comment:
what do these extra options do?

Reply:
Hmm... I think they might not be needed. They are tags which are not in use in our repo. I broke something in an earlier iteration and tried to make the run command identical to what Jenkins used, but hoping it isn't needed 👀